src/commands/insult.ts

```diff
@@ -0,0 +1,69 @@
+import { Client, Message } from 'discord.js';
+import axios from 'axios';
```

Comment:
Does axios not have a properly exported es6 module to import? Or type definitions? The curly braces above mean that those are JS modules being individually brought in, so I'm just curious. If it doesn't then whatevs

Reply:
A quick glance at the npm package for axios shows a bunch of exported interfaces, but no classes to be used.